app/src/main/java/com/github/kulebin/myfoursquareapp/useCase/ShowDetailVenueUseCase.java

```diff
@@ -27,7 +27,7 @@ public void onStart() {
             public void onSuccess(final Venue venue) {
                 mPresenter.setProgress(false);
                 if (venue != null) {
```

Comment:
Now I check null in DataLoader class and in case null an OnError callback will be called